src/components/Navbar/SearchDrawer/PreInput/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Button, { ButtonSize, ButtonType } from 'src/components/dls/Button/Button';
+import { getSurahNavigationUrl } from 'src/utils/navigation';
+import AdvancedSearchLink from '../AdvancedSearchLink';
+import styles from './PreInput.module.scss';
+
+interface Props {
+  setSearchQuery: (searchQuery: string) => void;
+}
+
+const POPULAR_SEARCH_QUERIES = { Mulk: 67, Noah: 71, Kahf: 18, Yaseen: 36 };
```

Comment:
this is ok, what I had in mind is that the value will be a search result object. With resultType, id, etc. so that in the future we can support non surah searches.

This is an early optimization though. What we have is good :) no need to change it.